### ISSUE TITLE
Add combobox to select datacenter and infrastructure

### ISSUE THREAD

**Issue by rezib** [rackslab/RacksDB#38]
In datacenters and infrastructures views, the search input should be replaced with a combobox web component for more accessibility and easier navigation with keyboard. For example :

https://tailwindui.com/components/application-ui/forms/comboboxes

In its initial state, the combobox input should be empty and the  combobox panel should be open with all suggestions. When a value is selected, the router should open the corresponding route immediately.
